pyvista/plotting/utilities/sphinx_gallery.py

```diff
@@ -12,6 +12,8 @@ def _get_sg_image_scraper():
     'matplotlib' and 'mayavi'. Details on this implementation can be found in
     `sphinx-gallery/sphinx-gallery/494`_
 
+    This must be at the top level namespace of PyVista.
```

Comment:
Might prevent a double-take if we rephase to something like
```suggestion
    This must be imported into the top level namespace of PyVista.
```